packages/web-app-admin-settings/src/components/Users/UsersList.vue

```diff
@@ -227,8 +231,13 @@ export default defineComponent({
         let a, b
 
         if (prop === 'role') {
-          a = user1.role?.displayName || ''
-          b = user2.role?.displayName || ''
+          a = this.$gettext(
```

Comment:
You could use
```suggestion
          a = this.getRoleDisplayName(user1)
```
here and the same for `b` with `user2` down below.
```suggestion
          a = this.$gettext(
```